packages-web/calendar/tests/TestProjects/Mendix7/javasource/administration/proxies/Account.java

```diff
@@ -38,7 +38,7 @@
 			metaName = s;
```

Comment:
All proxies should be excluded, they are generated based on the model
In general all javasources can be excluded, unless you have java actions for the project, then only include those

Reply:
Removed all unnecessary files and folders from the test projects